src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -85,26 +84,67 @@ public Task RunAsync(ActionRunStage stage)
                 actionRunner.Stage = stage;
                 actionRunner.Condition = aStep.Condition;
 
-                var step = ExecutionContext.RegisterNestedStep(actionRunner, inputsData, location, Environment);
-
+                var step = ExecutionContext.RegisterNestedStep(actionRunner, inputsData, Environment);
```

Comment:
should we rename `RegisterNestedStep` to something like `CreateCompositeStep`?

Reply:
Sure